joomla/joomla-cms
# [4.3] guided tours module count

### Summary of Changes

Changes the way tours are counted in the module. The result is the same but without the warning about unreachable code

### Testing Instructions
Check to make sure that there are a maximum of 7 tours in the module dropdown
Change the count in the module and check that the dropdown is changed correctly



Please select:
